spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2849,33 +2825,23 @@ object QueryPlanSerde extends Logging with CometExprShim {
   }
 
   /**
-   * Whether the given Spark partitioning is supported by Comet.
+   * Whether the given Spark partitioning is supported by Comet native shuffle.
    */
-  def supportPartitioning(
-      inputs: Seq[Attribute],
-      partitioning: Partitioning): (Boolean, String) = {
-    def supportedDataType(dt: DataType): Boolean = dt match {
-      case _: ByteType | _: ShortType | _: IntegerType | _: LongType | _: FloatType |
-          _: DoubleType | _: StringType | _: BinaryType | _: TimestampType | _: DecimalType |
-          _: DateType | _: BooleanType =>
-        true
-      case _ =>
-        // Native shuffle doesn't support struct/array yet
```

Comment:
yes, it does! This method is removed and we now have a single `supportedShuffleDataType` method that is used for both native and columnar shuffle type checks.

Reply:
Thanks for that, I was so confused about having this supported check in at least 3 places